README.md

```diff
@@ -40,18 +40,76 @@ API_TOKEN = "<your api token>"
 
 ### Running the script
 
-1. Activate the virtual environment
+Run the following commad after activating the virtual environment.
 
 ```
-//for windows
-virtualenv/Scripts/Activate.ps1 //In Powershell
+cd experiments
+python main.py
+```
+
+### Running FastAPI server
+
+This command will run fastapi development server on localhost:8000
 
-//for linux/ Mac
-source virtualenv/bin/activate
 ```
+fastapi dev server.py
+```
+
+## API Endpoints
+
+The following API endpoints have been implmented.
+
+### Request
 
-2. Run the following commad
+`GET /`
 
 ```
-python main.py
+curl -X 'GET' \
+  'http://127.0.0.1:8000/' \
+  -H 'accept: application/json'
+```
+
+### Response body
+
+```
+{
+  "Message": "welcome to bons.ai"
+}
+```
+
+### Request
+
+`POST /prompt`
+
+```
+curl -X 'POST' \
+  'http://127.0.0.1:8000/prompt' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "question": "what is my name",
+  "context": "my name is aaroosh",
+  "model": "distilbert-base-cased"
+}'
+```
+
+In this POST request we need to send request body having the following structure
+
+```
+{
+  "question": string,
+  "context": string,
+  "model": "distilbert-base-cased" or "roberta-base-squad2" or "google-bert" or "electra_large_discriminator"
+}
+```
+
+### Response body
+
+```
+{
+  "score": 0.9413116574287415,
+  "start": 11,
+  "end": 18,
+  "answer": "aaroosh"
+}
```

Comment:
These are a great addition to the Readme. Did you know also when start the fastapi server trying going to the localhost:8000/docs in your browser (or might be #docs)

Reply:
This is called an OpenAPI document and is the way that people now days share the REST API Servers they develop (at the top of the page there should be a link to the openapi.json)

Reply:
Ok I will modify the readme to inlcude openapi.json